CONTRIBUTING.md

```diff
@@ -860,19 +857,19 @@ sponsors:
       ---
       ```
 
-    - `new-url` can be either an internal relative URL ("/admin/config/tool-dependencies/", note the missing `src` at the front and the missing `index.md` at the end.) or point outside the hub.
+    - `new-url` can be either an internal relative URL ("/admin/config/tool-dependencies/", note the missing `src` at the front and the missing `index.md` at the end) or point outside the hub.
     - You are encouraged to also drop all text from the page.  Optionally, you can include something like:
 
       ```
       This tutorials index page has been merged with the new [Learn Galaxy](/learn/)
       hub page, please check there for up-to-date references to tutorials.
       ```
 
-- :chicken: When deleting, consolidating, or renaming pages please save the urls by adding redirects to the [rewrite file](https://github.com/galaxyproject/infrastructure-playbook/blob/master/galaxyenv/templates/nginx/galaxyproject.j2#L46) in the infrastructure playbook.
+- :chicken: When deleting, consolidating, or renaming pages, please save the urls by adding redirects to the [rewrite file](https://github.com/galaxyproject/infrastructure-playbook/blob/master/galaxyenv/templates/nginx/galaxyproject.j2#L46) in the infrastructure playbook.
```

Comment:
This link is broken and I'm not sure where the file is now or whether this line makes sense anymore

Reply:
It is this one here: https://github.com/galaxyproject/infrastructure-playbook/blob/master/env/galaxy/templates/nginx/galaxyproject.j2

Reply:
danke sehr!